joomla/joomla-cms
# [4.0] Call namespace helper classes

Pull Request for Issue # .

### Summary of Changes
In Joomla 4.0, we namespaced all of our components. However, for helper classes, we had to keep these classes none-namespace because these classes have methods use by other components.

As a result, we have both namespace and none-namespace helper classes in the same component, it is not nice.

This PR shows an idea to solve it. If an extension has namespace helper class, this namespace class will be used, otherwise, the none namespace class will be used.

For this one, I starts with CategoriesHelper class (addSubmenu method), Categories model and Tags model (method countItems)

### Testing Instructions
I am not sure if this will be accepted, so for now, open for discussion only. If the idea is accepted, just test banners component (backend) and make sure it is still working
